src/plugins/ExpressPlugin.ts

```diff
@@ -95,6 +94,8 @@ class ExpressPlugin implements SwPlugin {
       } catch (e) {
         stopIfNotStopped(e);
         throw e;
+      } finally {  // req.protocol is only possibly available after call to _handle()
+        span.tag(Tag.httpURL(((req as any).protocol ? (req as any).protocol + '://' : '') + (req.headers.host || '') + req.url));
```

Comment:
You moved this, is this the reason?

Reply:
Yes, because that field `req.protocol` is not available until after the call to `_handle()`.

Reply:
> Yes, because that field `req.protocol` is not available until after the call to `_handle()`.

I meant this is the reason why the tag orders changed. Is it?

Reply:
:facepalm: :facepalm: :facepalm: